content/liveobjects/storage.textile

```diff
@@ -0,0 +1,56 @@
+---
+title: Object storage
+meta_description: "Learn about LiveObjects object storage."
+product: liveobjects
+languages:
+  - javascript
+---
+
+<aside data-type='experimental'>
+<p>LiveObjects is currently Experimental. Its features are still in development and subject to rapid change.</p>
+</aside>
+
+h2(#default-persistence). Default object storage - 90 days
+
+Ably durably stores all objects on a channel for 90 days by default. If the data is not updated within 90 days, it will automatically expire. After expiry, the channel is reset to its initial state and only includes an empty "root object":/docs/liveobjects/concepts/objects#root-object.
+
+<aside data-type='note'>
+<p>In a future release you will be able to configure the default object storage duration via "channel rules":/docs/channels#rules . If you're interested in this feature, please "get in touch":/contact with your requirements.</p>
+</aside>
```

Comment:
I normally push back on statements like this. IMO docs should refer to the state of play as of today, rather than a theoretical future release.

Reply:
Thanks, what's your reasoning for that approach?

My intention with this is to a) communicate that the shortcoming is known and will be addressed and b) include a call to action to capture more information from anyone who has a need for this feature, as that is useful feedback for us to consider.

Reply:
Normally because even with the best intentions, releases or products can get de-prioritised. We then have statements promising enhancements or new features in a future release that sit and gather dust.

I'm certainly not against publishing a public roadmap, for example, but if I'm reading about functionality today I'd rather know what exists in the current state of play - especially if there isn't a specific timeline involved.

Reply:
Okay thanks that's fair enough, I've added [54d692d](https://github.com/ably/docs/pull/2591/commits/54d692d0d901ddb9de63b305b76e27d8adac64c7) which changes the language to encourage contacting us without making explicit roadmap promises, hopefully this is a good compromise.

Reply:
Yep, seems sensible to me.